src/agent/plugins/build/sourceMappings.ts

```diff
@@ -0,0 +1,244 @@
+/// <reference path="../../definitions/shelljs.d.ts"/>
+/// <reference path="../../definitions/Q.d.ts" />
+
+import Q = require('q');
+import agentifm = require('vso-node-api/interfaces/TaskAgentInterfaces');
+import shell = require('shelljs');
+import cm = require('../../common');
+import utilm = require('../../utilities');
+import crypto = require('crypto');
+import path = require('path');
+import fs = require('fs');
+import tm = require('../../tracing');
+
+//
+// Structure for persisting source mapping information to disk
+// See: https://github.com/Microsoft/vso-agent/blob/master/docs/workfolder.md
+// 
+// all paths are relative making moving working directories less friction
+//
+export interface ISourceMapping {
```

Comment:
The Windows agent has a property "fileFormatVersion: 2" which it uses to distinguish between the old file format. Should this have the same?